R/slice_derivation.R

```diff
@@ -0,0 +1,171 @@
+#' Execute a Derivation with Different Arguments for Subsets of the Input Dataset
+#'
+#' The input dataset is split into subsets (slices) and for each slice the
```

Comment:
Think line 3 should read `The input dataset is split into slices (subsets) and for each slice the`

Reply:
Updated